packages/clay-management-toolbar/src/ClayManagementToolbar.js

```diff
@@ -77,14 +77,32 @@ class ClayManagementToolbar extends ClayComponent {
 	}
 
 	/**
-	 * Continues the propagation of the plus button clicked event
+	 * Continues the propagation of the plus button clicked event. This path only
+	 * happens when there's a single primary item. Thus, the data payload associated
+	 * with the event is simply the first primary item.
 	 * @param {!Event} event
 	 * @private
 	 * @return {Boolean} If the event has been prevented or not.
 	 */
 	_handleCreationButtonClicked(event) {
+		const creationMenu = this.creationMenu;
+		const primaryItems = creationMenu.primaryItems;
```

Comment:
hey @jbalsas we have to do a better check here, this event is also triggered when we do not have a `creationMenu`.

https://github.com/liferay/clay/blob/2.x-develop/packages/clay-management-toolbar/src/ClayManagementToolbar.soy#L501

Reply:
good catch! I assumed we wouldn’t be firing this if we didn’t have a menu... could you please fix it and resend this?

Thanks!